lambda/index.py

```diff
@@ -34,7 +35,42 @@ class SubscriptionArgs:
     role_arn: str
 
 
-def modify_subscription(client, is_create: bool, log_group_name: str, subscription_args: SubscriptionArgs) -> bool:
+class AWSWrapper:
```

Comment:
An alternative to a wrapper class:

https://github.com/spulec/moto

I haven't used this before. The coverage is limited and probably doesn't address everything you need, particularly things outside of boto3 (cfnresponse). Just sharing.

Reply:
I think for now I won't do anything. I haven't set up enough Python projects recently to try to deal with 3rd dependencies yet.

Interesting to see some interpreted code options after working with Go so much.